src/components/MovieDetails/index.tsx

```diff
@@ -122,6 +122,9 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
 
   if (
     data.mediaInfo?.plexUrl4k &&
+    hasPermission([Permission.REQUEST, Permission.REQUEST_MOVIE], {
+      type: 'or',
+    }) &&
     hasPermission([Permission.REQUEST_4K, Permission.REQUEST_4K_MOVIE], {
       type: 'or',
     })
```

Comment:
Should we require base request permissions to be able to request 4k?

Reply:
@sct We actually already do. This is simply fixing the behavior in places where we don't currently check for the base request perms.

Reply:
(I would not be opposed to modifying this behavior, however.)

Reply:
Yeah I mean now that we have the ability to select movies/tv for both types, one use case might be ONLY allowing 4k movies/tv, but no standard requests?